.travis.yml

```diff
@@ -7,10 +7,10 @@ matrix:
   include:
   - jdk: openjdk8
     scala: 2.12.8
-    env: SPARK_VERSION="2.4.0" INTEGRATION_TESTS="true" "SNOWFLAKE_TEST_ACCOUNT"="aws" "SPARK_CONN_ENV_USE_COPY_UNLOAD"="false"
+    env: SPARK_VERSION="2.4.0" INTEGRATION_TESTS=true SNOWFLAKE_TEST_ACCOUNT=aws SPARK_CONN_ENV_USE_COPY_UNLOAD=false
```

Comment:
Why do we not use the quotes anymore?

Reply:
It looks this will cause the travis build failure. We didn't see the failure before.